src/site/_collections/authors.js

```diff
@@ -71,56 +71,68 @@ module.exports = (collections) => {
   /** @constant @type {Object.<string, Author>} @default */
   const authors = {};
 
-  Object.values(contributors)
-    .sort((a, b) => a.title.localeCompare(b.title))
-    .forEach((author) => {
-      // This updates the shared contributors object with meta information and is safe to be called multiple times.
-      author.url = path.join('/en', author.href);
-      author.data = {
-        title: author.title,
-        subhead: author.description,
-      };
-
-      author.elements = authorsPosts.has(author.key)
-        ? authorsPosts.get(author.key)
-        : [];
-
-      // If the author doesn't have any posts, use their Twitter profile.
-      if (author.elements.length === 0) {
-        if (!author.twitter) {
-          // If we're screenshot testing just ignore it.
-          if (process.env.PERCY) {
-            return;
-          }
-
-          // posts.length might be empty if we're generating partials so only
-          // log a warning if it has a value and we're doing a regular build.
-          // Note this is checking for _all_ posts and not just the posts
-          // by this specific author.
-          if (posts.length) {
-            console.warn(
-              `author ${
-                author.title
-              } has no posts and no social: ${JSON.stringify(author)}\n`,
-            );
-          }
-        } else {
-          author.href = `https://twitter.com/${author.twitter}`;
+  Object.keys(contributors).forEach((key) => {
```

Comment:
I think this seems pretty chill and we can go with this approach instead of the interstitial file i suggested